.editorconfig

```diff
@@ -3,21 +3,8 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
-indent_size = 2
+indent_size = 4
```

Comment:
This should definitely be 2, so that String doesn't get reformatted. 

Does tab_width need to be set if tabs arent used?

Probably try editing a bunch of files and make sure your configuration in this file doesn't try to rewrite other files in the repo.

Reply:
Ok THX